# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-properties-values-api/animation/custom-property-animation-transform-none.tentative.html.ini
# Commit: 6f62586a139b
# Full Hash: 6f62586a139b1fcc4817dddd924400958d89a6bb
# Author: Zach Hoffman <zach@zrhoffman.net>
# Date: 2023-12-18 17:17:57
# Regressor Bug: 1846516
# File Overlap Count: 2
# Description:
#   Bug 1846516 - [css-properties-values-api] Animate custom properties in a discrete way. r=firefox-style-system-reviewers,emilio
#   
#   This patch also updates the bug ID for a FIXME leftover from bug 1840478
#   to bug 1869476, since the same FIXME is added in D190758.
#   
# ==============================================================================

diff -r 729abedf9bda -r 6f62586a139b testing/web-platform/meta/css/css-properties-values-api/animation/custom-property-animation-transform-none.tentative.html.ini
--- a/testing/web-platform/meta/css/css-properties-values-api/animation/custom-property-animation-transform-none.tentative.html.ini	Mon Dec 18 09:24:08 2023 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,6 +0,0 @@
-[custom-property-animation-transform-none.tentative.html]
-  [Animating a custom property of type "<transform-list>|none" from "none" to <transform-list> value]
-    expected: FAIL
-
-  [Animating a custom property of type "<transform-function>|none" from "none" to <transform-function> value]
-    expected: FAIL